src/contracts/gho/GhoToken.sol

```diff
@@ -69,7 +72,11 @@ contract GhoToken is ERC20, Ownable, IGhoToken {
     address facilitatorAddress,
     string calldata facilitatorLabel,
     uint128 bucketCapacity
-  ) external onlyOwner {
+  ) external {
+    require(
+      hasRole(DEFAULT_ADMIN_ROLE, msg.sender) || hasRole(FACILITATOR_MANAGER, msg.sender),
```

Comment:
We can simplify this and check for the specific admin only imo. If the DEFAULT_ADMIN_ROLE wanna be manager, they can always grant the role themselves.